### DISCUSSION TITLE
Sponsorship?

### DISCUSSION THREAD

**Discussion by RagingRoosevelt** [dune3d/dune3d#90] [General]
Can project sponsorship be enabled? I'm already sponsoring FreeCAD and OpenSCAD and i think it's important to sponsor any project that is working on open source parametric CAD in order for the hobbiest community to have viable alternatives to fusion and it's ilk.

**Comment by carrotIndustries** [comment]
So far, I'm too lazy do deal with the tax obligations that come with receiving sponsor payouts.

I also want to keep money out of the equation to not create any perceived obligations on my end.

**Reply by wucke13** [reply]
@carrotIndustries Just a side note: consider talking to a tax attorney, I long felt similar to you, but came to realize that the effort can be kept minimal if one prefers to remain lazy (and does not strive to use any trick in the book to lower taxes) and that sponsoring/donations for open source work can be quite simple as long as it stays below some threshold, (I believe 8k€ yearly) for Germany.

This is by no means a "but you should ...", just a note of "have been there, have done that (being lazy about it)" but eventually I learned that things are not as laborious as I anticipated.

**Comment by JaneSmith** [comment]
I've been looking at alternatives to FreeCAD lately, and discovered Dune 3D. I've seen SolveSpace, but I've never come across this app before. I just wanted to let you know that I am loving this app so much. It's such a breath of fresh air. I love the clean, minimal, GNOME-like interface, with the command palette to quickly access all the functions. This is such a joy to use. Just wanted to share some positive feedback, but didn't want to clog up the discussion board so this seemed like an appropriate place to post... Thank you for making this!
